integration/rules/javascript_test.go

```diff
@@ -2,122 +2,96 @@ package integration_test
 
 import "testing"
 
+const javascriptRulesPath string = "../../pkg/commands/process/settings/rules/javascript/"
+
 func TestJavascriptLangLoggerSummary(t *testing.T) {
```

Comment:
Suggestion to remove `Summary` from the naming to avoid possible confusion

```suggestion
func TestJavascriptLangLogger(t *testing.T) {
```

Reply:
done